components/SpotlightShare/icon.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
```

Comment:
Again, and I can't stress this enough ... unless it's a restriction by a library (like it is with Modal), use `NEXTJS' css-in-js` ... I'll share the link again https://nextjs.org/docs/basic-features/built-in-css-support#css-in-js Please replace `styled-components` with that.